doc/man3/RAND_DRBG_set_callbacks.pod

```diff
@@ -99,9 +101,9 @@ RAND_DRBG_set_callbacks() return 1 on success, and 0 on failure
 
 =head1 NOTES
 
-It is important that B<cleanup_entropy>() and B<cleanup_nonce>() clear the buffer
-contents safely before freeing it, in order not to leave sensitive information
-about the DRBG's state in memory.
+It is important that B<cleanup_entropy>() clear the buffer contents safely
```

Comment:
While it's not required by the standard, I would still erase the nonce after use, just as we do it with entropy and additional data. So maybe just leave the formulation as it is?